exchange/exchange.go

```diff
@@ -637,19 +649,20 @@ func (e *exchange) makeExtBidResponse(adapterBids map[openrtb_ext.BidderName]*pb
 		}
 	}
 
-	for a, b := range adapterBids {
-		if b != nil && req.Test == 1 {
+	for a, b := range adapterExtra {
```

Comment:
Consider renaming variables for clarity. Something along the lines of:
```
for bidderName, reasponseExtra := range adapterExtra
```

I * think * before `a` was for adapter and `b` was for bids. Since we're changing the data types might be a good opportunity to make these variable names clearer.

Reply:
Done!